### DISCUSSION TITLE
element added to canvas does not have auto generated id

### DISCUSSION THREAD

**Discussion by karol93** [GrapesJS/grapesjs#5208] [Q&A]
element added to canvas does not have an auto-generated id.
the id is added after I select the element on the canvas. (when i click on this element when it is already on the canvas)
is there a possibility of adding an id to the element without selecting it?

i see that component.view has id but i do not see id in html after call getHtml()

**Comment by bgrand-ch** [comment]
@karol93 Hello,

More informations please, with screenshots and demo/current code.
